tasks/arrays/arrays.js

```diff
@@ -35,6 +44,7 @@ function getJerryInfo(chars) {
  * @return {boolean}
  */
 function isAllHuman(chars) {
+    return (chars.every(element => element.species == 'Human'));
```

Comment:
() brackets can be removed before and after the statement
return chars.every(element => element.species == 'Human');